pages.zh/common/go.md

```diff
@@ -0,0 +1,33 @@
+# go
+
+> 管理 Go 源代码的工具。
+> 一些子命令，例如 `go build` 有自己的使用文档。
```

Comment:
Follow [subcommand-mention.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#zh).
```suggestion
> 此命令也有关于其子命令的文件，例如：`go build`.
```

Reply:
> Follow [subcommand-mention.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#zh).

Thanks for reminding. I'll check out these docs next time.